examples/yacat/yacat.py

```diff
@@ -20,146 +24,184 @@
     TEXT_COLOR_YELLOW,
 )
 
+HASHCAT_ATTACK_MODE = 3  # stands for mask attack, hashcat -a option
+KEYSPACE_OUTPUT_PATH = Path("/golem/output/keyspace")
 
-def write_hash(hash):
-    with open("in.hash", "w") as f:
-        f.write(hash)
+# Ideally, this value should depend on the chunk size
+ATTACK_TIMEOUT: timedelta = timedelta(minutes=30)
```

Comment:
maybe `DICTIONARY_ATTACK_TIMEOUT` ? ... just "attack" seems more serious than it is to a casual (non-security-terminology-aware) user ...

Reply:
Makes sense, though using the word dictionary in this case may be confusing (i.e. in `hashcat` dictionary and mask are two different types of attacks). How about `MASK_ATTACK_TIMEOUT` and renaming the worker funcion to `perform_mask_attack`?